src/compound/SupplyVault.sol

```diff
@@ -137,7 +115,7 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
     }
 
     function _accrueUnclaimedRewards(address _user) internal returns (uint256 unclaimed) {
-        return _accrueUnclaimedRewardsFromRewardsIndex(_user, _claimRewards());
+        return _accrueUnclaimedRewardsFromRewardsIndex(_user, _claimVaultRewards());
     }
 
     function _accrueUnclaimedRewardsFromRewardsIndex(address _user, uint256 _newRewardsIndex)
```

Comment:
Perhaps we should skip this when `_user == address(0)`. Else, for each mint/burn we would run the logic.

Reply:
Yes, and it is expected as we want to accrue unclaimed rewards when depositing and withdrawing